kubernetes/zeus/zeus/web.yaml

```diff
@@ -1,3 +1,42 @@
+apiVersion: networking.gke.io/v1beta1
+kind: ManagedCertificate
+metadata:
+  name: zeus-ci
+  namespace: default
+  labels:
+    app: web
+spec:
+  domains:
+    - zeus.ci
+    - updates.zeus.ci
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: web-gce
```

Comment:
current ingress is 'web' - i accidentally deleted it from the repo without adding it to this combined yaml